### PR TITLE
IEv11 friendly

### PR DESCRIPTION
**Pull request by ericminio**

### CODE REVIEW

File: public/css/open-iconic-bootstrap.css [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 @font-face {
   font-family: 'Icons';
-  src: url('../fonts/open-iconic.eot');
-  src: url('../fonts/open-iconic.eot?#iconic-sm') format('embedded-opentype'), url('../fonts/open-iconic.woff') format('woff'), url('../fonts/open-iconic.ttf') format('truetype'), url('../fonts/open-iconic.otf') format('opentype'), url('../fonts/open-iconic.svg#iconic-sm') format('svg');
+  src: url('../fonts/open-iconic.woff') format('woff');
```
Comment (perkinss) [inline public/css/open-iconic-bootstrap.css]: It makes me sad to say farewell to font-awesome.

File: src/components/Form2DataSection.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ class Form2DataSection extends React.Component {
                         <div className="row proceeding-style">
                             <div className="col-lg-2 col-md-2 col-sm-6 col-xs-6 respondent-name-label">
                                 <div style={{whiteSpace: 'nowrap'}}>
-                                    Respondent name(s) &nbsp;
-                                    <i className="fa fa-question-circle" aria-hidden="true" data-tip="Select the names of all those responding to the appeal"></i>
+                                    Respondent's name &nbsp;
```
Comment (perkinss) [inline src/components/Form2DataSection.js]: You overwrote my change here.

File: src/components/Journey.js [hunk 3]
```diff
@@ -33,9 +33,9 @@ class Journey extends Component {
         this.leaveToAppealRefused = this.leaveToAppealRefused.bind(this);
         this.handlePopState = this.handlePopState.bind(this);
         this.dataTip = `<p>If you don't know whether you have the right to appeal,</p>
-                        <p>please see our online guidebook for more 
+                        <p>please see our online guidebook for more
                         <u><a href='https://www.courtofappealbc.ca/appellant-guidebook/1.2-do-you-have-a-right-to-appeal-your-case'
-                        target="_blank">
+                        target="_blank" rel="noopener noreferrer">
```
Comment (perkinss) [inline src/components/Journey.js]: good catch!

File: src/components/Form2DataSection.js (hunk 2 above)
Comment (ericminio) [reply src/components/Form2DataSection.js]: changing back to "Respondent name(s) &nbsp;"...

File: src/helpers/find.case.by.id.js [hunk 4]
```diff
@@ -1,8 +1,15 @@
 let findCaseById = function(options, then) {
-    let found = options.cases.find((item) => item.id === options.id );
```
Comment (perkinss) [inline src/helpers/find.case.by.id.js]: seriously? IE doesn't support this? :(

Comment (ericminio) [reply src/helpers/find.case.by.id.js]: you see my fun?

Comment (perkinss) [reply src/helpers/find.case.by.id.js]: I think you still might be able to use javascript map reduce, no?

Comment (ericminio) [reply src/helpers/find.case.by.id.js]: https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/find#Compatibilit%C3%A9_des_navigateurs

File: src/components/Form2DataSection.js (hunk 2 above)
Comment (ericminio) [reply src/components/Form2DataSection.js]: new commit added

File: src/helpers/find.case.by.id.js (hunk 4 above)
Comment (perkinss) [reply src/helpers/find.case.by.id.js]: en français!

Comment (perkinss) [reply src/helpers/find.case.by.id.js]: so much fun

**Pull request merged**